packages/phone-number-privacy/signer/test/integration/domain.test.ts

```diff
@@ -476,16 +486,21 @@ describe('domainService', () => {
           now: res1.body.status.now,
         },
       })
+      const eval1 = thresholdPoprfClient.unblindPartialResponse(
+        Buffer.from(res1.body.signature, 'base64')
+      )
+      expect(eval1.toString('base64')).toEqual(expectedEval)
 
       // submit identical request with nonce set to 1
-      const req2 = (await signatureRequest(undefined, 1))[0]
+      req1.options.nonce = defined(1)
+      req1.options.signature = noString
```

Comment:
Similar to the above but slightly more detail: if we don't reset it to `noString`, the signature is over the request (including the old signature), which then doesn't match (since we don't end up sending the old sig in the request). The EIP712 data structure that gets signed (and that the server-side reconstructs) has the options.signature field (i.e. can't delete it outright -- we will get an error since the field needs to be defined even if it doesn't have a value), but it also expects the field to be empty hence the `NoString`. Hope that makes sense!!